controllers/topology/reconciler.go

```diff
@@ -83,8 +83,8 @@ func (r *Reconciler) ReconcileDeployments(
 	}
 
 	for _, nodeName := range nodesNeedingRestart {
-		if !clabernetesutil.StringSliceContains(deployments.Missing, nodeName) {
-			// is a new node, don't restart
+		if clabernetesutil.StringSliceContains(deployments.Missing, nodeName) {
+			// is a new node, don't restart, we'll deploy it soon
```

Comment:
might be the right time to swap this for a stdlib https://pkg.go.dev/slices#Contains

Reply:
done!